app/controllers/campaigns_controller.rb

```diff
@@ -22,6 +22,11 @@ def show
   # POST /campaigns
   def create
     @campaign = Campaign.create!(create_params)
+    if params[:has_square_fee]
+      @fee = Fee.find(1)
+      @campaign.fees << @fee
+      @fee.campaigns << @campaign
```

Comment:
Actually wasn't able to find if this automatically works both ways for has_and_belongs_to_many relationships, I'll write a test to confirm this